### PR TITLE
Add validation for the LoginView inputs

### PR DESCRIPTION
**Pull request by Shadic78**
Co-authored-by: EusebioAjas <eusebioajas6@gmail.com>

# Description
What did you implemented/Why did you implemented this:
  - What?: I've added react native form validator
 - Why?: To validate the inputs of the Login view
  
  ## Screenshots 
image

### CODE REVIEW

File: src/components/ErrorMessage/ErrorMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  return(
+    <View style={styles.container}>
+      {errorMessages && errorMessages.map((message, index) => (
+        <Text key={index} style={styles.label}>*{message}</Text>
+      ))}
+    </View>
+  );
+};
+
+export default ErrorMessage;
```
Comment (DemisRincon) [inline src/components/ErrorMessage/ErrorMessage.jsx:15]: este comopente esta talvez un poco de mas a menos que vayas a utilizar el mensaje de error en algo que no sea un input

File: src/screens/LoginView/LoginView.jsx [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
+  };
+
   return(
     <ScrollView style={styles.container}>
       <Text style={styles.title}>Sign In</Text>
       <View style={styles.inputContainer}>
-        <Text style={styles.textInputLabel}>Email</Text>
+        <InputLabel errorMessages={getErrorsInField('email')}>Email</InputLabel>
         <TextBox style={styles.textInputLabel} value={email} onChange={setEmail} />
       </View>
```
Comment (DemisRincon) [inline src/screens/LoginView/LoginView.jsx:32]: each of these blocks can be integrated as a component, it is possible that codacy or codelclimate mark them as repeated code

**Review by DemisRincon — APPROVED** [review]
Some commentaries that could improve your code but in general looks good

File: src/util/colors.js [hunk 3]
```diff
@@ -3,4 +3,5 @@ export const colors = {
   lightGray: '#4d4d4d',
   white: '#fff',
```
Comment (jkzasori) [inline src/util/colors.js:4]: If in the future the app's color change you have to change each part where you put this colors, so I think is better to have names for colors like primary-color, secondary-color, acent-color, or something like that.

**Review by jkzasori — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]

**Pull request merged**